application.py

```diff
@@ -160,5 +160,5 @@ def check_login_creds():
 
 # Run server
 if __name__ == '__main__':
-    application.run(host='0.0.0.0') 
-    # application.run(debug=True)
+    #application.run(host='0.0.0.0') 
+     application.run(debug=True)
```

Comment:
If we are merging this to main we should have line 164 commented out and have line 163 active for deployment